src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,124 @@
+statements_in:
+  type: object
+  properties:
+    file_id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: fidftadfdsdf
+  required:
+    - file_id
+
+statements_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtwownwrng
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: string
+      description: |
+        Specifies the size of file uploaded by the user.
+      example: 21KB
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the document uploaded by the user.
+      example: 3
+    no_of_credit:
+      type: integer
+      description: |
+        Number of credit transactions in the document uploaded by the user.
+      example: 3
+    no_of_transactions:
+      type: integer
+      description: |
+        Number of transactions in the document uploaded by the user.
+      example: 3
+    statement_columns:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies headers of the statement uploaded by the user.
+      example:  
+        - account_number
+        - amount
+    statement_mapping_id:
+      type: string
+      description: |
+        Refers to the mapping used in statement.
+      example: stmpdfddfd
+    start_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Start date of the statement period
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+    has_statement_duplicate:
+      type: boolean
+      description: |
+        Determines whether the statement is duplicate or not.
+    duplicate_statement_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_string'
+      example:
+        - stmtge49ielgel
+    has_statement_line_items_duplicate:
+      type: boolean
+      description: |
+        Determines whether the transaction inside the statement is duplicate or not.
```

Comment:
instead of `Determines` can use
indicates whether there are any line items in this statement are duplicates.